scripts/plutus/staking-example/register-stake-pool.sh

```diff
@@ -132,7 +132,7 @@ cardano-cli transaction build \
   --testnet-magic "$TESTNET_MAGIC" \
   --change-address "$utxoaddrwithstaking" \
   --tx-in "$keytxin" \
-  --tx-out "$utxoaddrwithstaking+1000" \
```

Comment:
How are the changes in this file related to the PR?

Reply:
I had to test the command with a stake pool locally. In order to do that I had to use this script but the output values were not large enough to satisfy the min utxo.